extensions/sql-database-projects/src/test/netCoreTool.test.ts

```diff
@@ -8,30 +8,34 @@ import * as os from 'os';
 import * as fs from 'fs';
 import * as path from 'path';
 import * as vscode from 'vscode';
+import * as sinon from 'sinon';
 import { NetCoreTool, DBProjectConfigurationKey, NetCoreInstallLocationKey, NextCoreNonWindowsDefaultPath } from '../tools/netcoreTool';
 import { getQuotedPath } from '../common/utils';
 import { isNullOrUndefined } from 'util';
 import { generateTestFolderPath } from './testUtils';
 
-describe.skip('NetCoreTool: Net core tests', function (): void {
+describe('NetCoreTool: Net core tests', function (): void {
 
 	it('Should override dotnet default value with settings', async function (): Promise<void> {
 		try {
 			// update settings and validate
 			await vscode.workspace.getConfiguration(DBProjectConfigurationKey).update(NetCoreInstallLocationKey, 'test value path', true);
 			const netcoreTool = new NetCoreTool();
+			sinon.stub(netcoreTool, 'showInstallDialog').returns(<any>Promise.resolve());
 			should(netcoreTool.netcoreInstallLocation).equal('test value path'); // the path in settings should be taken
-			should(netcoreTool.findOrInstallNetCore()).equal(false); // dotnet can not be present at dummy path in settings
+			should(await netcoreTool.findOrInstallNetCore()).equal(false); // dotnet can not be present at dummy path in settings
 		}
 		finally {
 			// clean again
 			await vscode.workspace.getConfiguration(DBProjectConfigurationKey).update(NetCoreInstallLocationKey, '', true);
+			sinon.restore();
```

Comment:
sinon.restore() can be in an afterEach() so that each test doesn't have to call it

Reply:
Yep, missed doing that. Updated it.